include/vkd3d_device_ext.idl

```diff
@@ -0,0 +1,38 @@
+/*
+ * * Copyright 2021 NVIDIA Corporation
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
+ */
+
+import "vkd3d_d3d12.idl";
+
+typedef enum D3D12_VK_EXTENSION
+{
+    D3D12_GET_EXTENSION_SUPPORT   = 0x1,
+
+} D3D12_VK_EXTENSION;
+
+[
+    uuid(11ea7a1a-0f6a-49bf-b612-3e30f8e201dd),
+    object,
+    local,
+    pointer_default(unique)
+]
+interface ID3D12DeviceExt : IUnknown
+{
+    HRESULT GetVulkanHandles(void* pInstance, void* pPhysDev);
```

Comment:
Pointer asterisk binds right

Reply:
Don't you need VkDevice as well?

Reply:
Please no `void *`. Just forward declare the handles for the idl and cpp_quote including vulkan/vulkan_core.h